app/models/company.rb

```diff
@@ -18,4 +18,15 @@ class Company < ActiveRecord::Base
                      "Confirmed" => "confirmed",
                      "Dead" => "dead"
                    }
+                   
+  def self.company_count_by_status(status)
+    self.send(status.to_sym).count
+  end
+
+  def self.company_count_by_city
+    self.order(:city)
+        .pluck(:city)
+        .group_by{ |city| city}
+        .inject({}) {|hash, (k,v)| hash[k] = v.size; hash}
+  end
```

Comment:
Love the use of send and inject!! :tada: 

This would work without the selfs on line 23, and 27. Not sure what is convention (may be good to keep for readability) but I know Hound used to always yell at me for including them. 

` def self.company_count_by_status(status)`
   `send(status.to_sym).count`
 `end`

` def self.company_count_by_city`
`    order(:city)`
`    .pluck(:city)`
`.group_by{ |city| city}`
`.inject({}) {|hash, (k,v)| hash[k] = v.size; hash}`
`end`